src/pint/models/tcb_conversion.py

```diff
@@ -0,0 +1,119 @@
+"""TCB to TDB conversion of a timing model."""
+
+import numpy as np
+
+from pint.models.parameter import MJDParameter
+
+__all__ = [
+    "IFTE_K",
+    "scale_parameter",
+    "transform_mjd_parameter",
+    "convert_tcb_to_tdb",
+]
+
+# These constants are taken from Irwin & Fukushima 1999.
+# These are the same as the constants used in tempo2 as of 10 Feb 2023.
+IFTE_MJD0 = np.longdouble("43144.0003725")
+IFTE_KM1 = np.longdouble("1.55051979176e-8")
+IFTE_K = 1 + IFTE_KM1
+
+
+def scale_parameter(model, param, n):
```

Comment:
Could you add a docstring?  It should also explain what this will do if the parameter is `None`

Reply:
Added more detailed docstring. `param` is supposed to be a string. It will throw an error if it is not a string.